### ISSUE TITLE
Add twitter trending section to home page or information page

### ISSUE THREAD

**Issue by GoOsTT** [zemlatruskavets/Ukraine_Aid_front#13]
This was an idea I had after reading the  comment section of your initial reddit post.
How would this page be more useful than social media? Well, it won't be, but could coexist very well in my opinion.

It would be a nice start to see updates on what's trending with hashtags around Ukraine.

**Comment by zemlatruskavets** [comment]
Yeah, this is a great idea. This way people could donate based on where there's a lot of activity on social media.
